src/implementations/twig/components/timeline/timeline.story.js

```diff
@@ -1,64 +1,75 @@
+import { loremIpsum } from 'lorem-ipsum';
 import { withNotes } from '@ecl/storybook-addon-notes';
-import { getExtraKnobs, tabLabels } from '@ecl/story-utils';
-import he from 'he';
 import withCode from '@ecl/storybook-addon-code';
+import { correctSvgPath } from '@ecl/story-utils';
 
-import defaultSprite from '@ecl/resources-ec-icons/dist/sprites/icons.svg';
 import demoData from '@ecl/specs-component-timeline/demo/data';
 import timeline from './timeline.html.twig';
 import notes from './README.md';
 
-const prepareTimeline = (data) => {
+const getArgTypes = () => {
+  const argTypes = {};
+  argTypes.showDummyContent = {
+    name: 'Add dummy content',
+    type: { name: 'boolean' },
+    defaultValue: false,
+    description: 'Add dummy content at the bottom of the timeline.',
+    table: {
+      category: 'Test content',
+    },
+    control: {
+      type: 'boolean',
+    },
+  };
+
+  return argTypes;
+};
+
+// Prepare dummy Html for the main content.
+const prepareHtmlContent = (state) => {
+  if (state) {
+    return `<p class="ecl-u-type-paragraph-m">${loremIpsum({ count: 25 })}</p>
+            <p class="ecl-u-type-paragraph-m">${loremIpsum({ count: 25 })}</p>`;
+  }
+
+  return '';
+};
+
+// Prepare data for the navigation.
+const prepareData = (data) => {
   const { from, to } = data.hide;
   let hiddenCount = 0;
   if (to > 0) {
     hiddenCount = to - from;
   } else {
     hiddenCount = data.items.length + to - from;
   }
-  data.toggle_collapsed = text(
-    'toggle_collapsed',
-    `Show ${hiddenCount} more items`,
-    tabLabels.required
-  );
-  data.toggle_expanded = text(
-    'toggle_expanded',
-    `Hide ${hiddenCount} items`,
-    tabLabels.required
-  );
-  data.icon_path = optionsKnob(
-    'icon_path',
-    { current: defaultSprite, 'no path': '' },
-    defaultSprite,
-    { display: 'inline-radio' },
-    tabLabels.required
-  );
-
-  data.hide.from = number('hide.from', data.hide.from, {}, tabLabels.optional);
-  data.hide.to = number('hide.to', data.hide.to, {}, tabLabels.optional);
-
-  data.items.forEach((item, i) => {
-    const { id, label, title, content } = item;
-    item.label = text(`items[${i}].label`, label, tabLabels.required);
-    item.id = text(`items[${i}].id`, id, tabLabels.optional);
-    item.title = text(`items[${i}].title`, title, tabLabels.optional);
-    item.content = he.decode(
-      text(`items[${i}].content`, content, tabLabels.required)
-    );
-  });
+  data.toggle_collapsed = `Show ${hiddenCount} more items`;
+  data.toggle_expanded = `Hide ${hiddenCount} items`;
 
-  getExtraKnobs(data);
+  data.icon_path = '/icon.svg';
+  correctSvgPath(data);
 
   return data;
 };
 
 export default {
   title: 'Components/Timeline',
+  parameters: {
+    knobs: {
+      disable: true,
+    },
+  },
 };
 
-export const Default = () => timeline(prepareTimeline(demoData));
+export const Default = (arg) => {
+  const timelineHtml = timeline(prepareData(demoData));
```

Comment:
It's fine eh, take this only as a suggestion, i personally don't like this string concatenation that we return by this function that forces you to return an empty string from the prepareHtmlContent when the boolean is false.
You could prepare the story inside the aforementioned function, instead, like this:
```
const prepareHtmlContent = (state) => {
  let story = timeline(prepareData(demoData));
  if (state) {
    story += `<p class="ecl-u-type-paragraph-m">${loremIpsum({ count: 25 })}</p>
            <p class="ecl-u-type-paragraph-m">${loremIpsum({ count: 25 })}</p>`;
  }

  return story;
};
```
and then you would export Default the normal way:

`export const Default = (arg) => prepareHtmlContent(arg.showDummyContent);`

Just a suggestion.
Another suggestion is: ES6 introduced a method for repeating strings, which is, predictably, `string.repeat(times)`
you could then add the "dummy content" like this:
```
    const dummy = `<p class="ecl-u-type-paragraph-m">${loremIpsum({ count: 25 })}</p>`;
    story += dummy.repeat(2);
```
but to be honest, do we really need two paragraphs..? :)

Reply:
Nice, Thks for suggestion 👍